Snowflake.Data.Tests/IntegrationTests/ConnectionMultiplePoolsIT.cs

```diff
@@ -313,7 +315,7 @@ public void TestNewConnectionPoolClean()
         [Test]
         public void TestConnectionPoolExpirationWorks()
         {
-            var connectionString = ConnectionString + "expirationTimeout=0;maxPoolSize=2";
+            var connectionString = ConnectionString + "expirationTimeout=0;maxPoolSize=2;minPoolSize=1";
```

Comment:
I could remove minPoolSize but it would be taken the default value of 2 for the minPoolSize.
Are you sure that we would like to test a feature of expiration for such a weird pool which has maxPoolSize = 2 and minPoolSize = 2. Adding minPoolSize = 1 makes this pool more normal.